symbulate/distributions.py

```diff
@@ -380,15 +492,33 @@ def __init__(self, mean, cov):
             raise Exception("The dimension of the mean vector" +
                             "is not compatible with the dimensions" +
                             "of the covariance matrix.")
-        self.mean = mean
-        self.cov = cov
+        
+        if len(mean) >= 2:
```

Comment:
I like your thinking, but I'm not sure I agree with this. You should be able to simulate a MultivariateNormal where the number of dimensions n=1. Of course, this is equivalent to just simulating a Normal.